woocommerce-gateway-stripe.php

```diff
@@ -24,8 +24,8 @@
  */
 define( 'WC_STRIPE_VERSION', '4.5.5' ); // WRCS: DEFINED_VERSION.
 define( 'WC_STRIPE_MIN_PHP_VER', '5.6.0' );
-define( 'WC_STRIPE_MIN_WC_VER', '3.0' );
-define( 'WC_STRIPE_FUTURE_MIN_WC_VER', '3.0' );
+define( 'WC_STRIPE_MIN_WC_VER', '3.3' );
+define( 'WC_STRIPE_FUTURE_MIN_WC_VER', '3.3' );
```

Comment:
We should probably put out one release before this with WC_STRIPE_MIN_WC_VER still at 3.0 and FUTURE at 3.3 to give merchants time to upgrade. And we should probably NOT roll WC_STRIPE_MIN_WC_VER until after the holidays. This might mean sitting on this PR until January.

What do you think, @v18 ?

Reply:
@allendav: so we need at least to ensure that we have `define( 'WC_STRIPE_MIN_WC_VER', '3.0' );` and revert line 27. Did I understand that correctly?

Reply:
Yes, I think so.... basically if this PR can proceed with WC_STRIPE_MIN_WC_VER 3.0 and we want to move to 3.3 in the near future (which we should honestly), I would set WC_STRIPE_FUTURE_MIN_WC_VER to 3.3 in this PR.

IMHO we should open a separate issue to bump WC_STRIPE_MIN_WC_VER to 3.3 as high priority after the holidays.

Reply:
Done in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1383/commits/978640d8f3f511aeef47832f05e1c379fd36afec

Reply:
We'll need a separate issue/PR for bumping `WC_STRIPE_MIN_WC_VER` in January+.

Reply:
Thanks for the tip! h/t @reykjalin
Opened an issue - https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1453